core/lib/permissions.ts

```diff
@@ -18,3 +18,39 @@ export const getPubUsers = (permissions: PermissionPayload[]) => {
 	}
 	return users;
 };
+
+enum Privileges {
+	movePub,
+	createPub,
+	viewPub,
+	updatePubValues,
+	createRelatedPub,
```

Comment:
It was the former, based on https://github.com/pubpub/platform/issues/595 saying that editors can create/edit related pubs. I think connecting an existing pub would fall under `updatePubValues`. But tbh, depending on how we implement it, it might not be different from `createPub` + `updatePubValues`. I expect to make some changes to this list pretty quickly once we have the actually write the queries for these.

Reply:
I think it makes sense to have separate capabilities for that then, i can see allowing someone to create a child pub/related pub, but not create any pub.